server/mdm/microsoft/wstep.go

```diff
@@ -104,7 +91,13 @@ func (m *manager) IdentityFingerprint() string {
 	return m.identityFingerprint
 }
 
-// TODO: Marcos to update the POC implementation of this function
+func (m *manager) IdentityCert() *x509.Certificate {
+	return m.identityCert
```

Comment:
Do we want to return the pointer to the original certificate or should we return a copy instead?

Reply:
better to return a copy, I've just fixed this